docs/source/index.rst

```diff
@@ -1,39 +1,91 @@
 ==========================
 FAIR Data Point API Client
 ==========================
+.. contents::
+    :depth: 3
 
 .. meta::
    :description: A collection of tutorials and guides for fairdatapoint-client package.
    :keywords: fairdatapoint, metadata, guide, tutorial, doc
 
-fairdatapoint-client is a simple and elegant Python package to work with REST
-APIs of FAIR Data Point.
+fairdatapoint-client is a simple and elegant Python package to work with
+REST APIs of `FAIR Data Point <https://github.com/fair-data/fairdatapoint>`_.
```

Comment:
Either "with _the_ FAIR Data Point REST API" or "with REST APIs of FAIR Data Points." I'm honestly not sure what the rule is and why it needs either the article or a plural here, but it does. And it's impossible if you're coming from a language without either articles or plurals :-(.